### ISSUE TITLE
Add wradlib content

### ISSUE THREAD

**Issue by mgrover1** [ProjectPythiaCookbooks/radar-cookbook#68]
This week at the Open Radar Science short course, @kmuehlbauer presented some awesome notebooks covering the wradlib basics and workflows
- [Rendered Jupyterbook](https://openradarscience.org/erad2022/wradlib/README.html)

with the content located here - https://github.com/openradar/erad2022/tree/main/notebooks/wradlib

**Comment by mgrover1** [comment]
@kmuehlbauer would you be interested in migrating over some of the wradlib content from that workshop? Planning on rebranding the landing page, replacing the ARM logo with the open radar logo since this should not be ARM specific.

**Comment by kmuehlbauer** [comment]
@mgrover1 Yes, that somehow fell through the cracks. Hope to get to it soon.

**Comment by kmuehlbauer** [comment]
I'll get a try on this now. I'll fetch the needed data from directly from https://github.com/openradar/erad2022 within the notebooks.

**Comment by kmuehlbauer** [comment]
waiting for https://github.com/wradlib/wradlib/issues/598

**Comment by brian-rose** [comment]
@mgrover1 just doing some triage of older issues. Is this still planned content work? It would be great if you can add a priority label for our project triage :-)

**Comment by mgrover1** [comment]
It is still planned work - I will add a tag 😄

**Comment by mgrover1** [comment]
Thanks @brian-rose !!

**Comment by kmuehlbauer** [comment]
@brian-rose Green looks definitely better that red. And yes, we should finally add these examples. I'm in the final stages of polishing Wradlib 2.0 and would wait here. So this might take another two months or so, if this is good with you.

**Comment by mgrover1** [comment]
^ Agreed! Sorry for that!

**Comment by brian-rose** [comment]
Ha ha I'm just in the middle of harmonizing the label colors across Pythia repos so I think the color is going to change one more time!
